enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java

```diff
@@ -246,7 +253,12 @@ public void memberIsFailed( InstanceId instanceId )
         {
             if ( !isQuorum( getAliveCount(), getTotalCount() ) )
             {
-                changeStateToPending();
+                String loggingMessage = "Got memberIsFailed(" + instanceId + ") and cluster lost quorum to continue";
```

Comment:
imho it is better to log things right here and not pass partial messages to changeStateToPending method